src/universal/modules/teamDashboard/components/TeamCallsToAction/LoadableTeamCallsToActionMenu.js

```diff
@@ -0,0 +1,17 @@
+// @flow
```

Comment:
By breaking it out into a loadable, webpack has the choice of chunking it with the team dash (which it will do because it's so small) or keeping it separate.
Not strictly necessary, but let's say this menu gets refactored & has to make a server request. With a loadable, we won't have to worry about any of that until the client hovers over the button, which means the initial view renders faster.